<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>general</class>
 <widget class="QDialog" name="general">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 128, 137);</string>
  </property>
  <widget class="QPushButton" name="networkingbut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:5px solid black;</string>
   </property>
   <property name="text">
    <string>Networking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about_but">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:none;
border:5px solid black;</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>801</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 128, 137);</string>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="seria_lab">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:2px solid black;
border-radius:20px;</string>
     </property>
     <property name="text">
      <string>Serial Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="software_lab">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 15pt &quot;Ubuntu Condensed&quot;;
font:bold;
border:2px solid black;
border-radius:20px;</string>
     </property>
     <property name="text">
      <string>Software Version</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color: rgb(255,255,255);
border-radius:20px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>340</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color: rgb(255,255,255);
border-radius:20px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>521</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Total Disk Space : 3599.00 MB&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Available Disk Space: 2997.00MB&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Log Space Utilization: 0.00MB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearallbut">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>370</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:2px solid black;</string>
     </property>
     <property name="text">
      <string>Clear All Logs</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>470</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QProgressBar{
	background-color: rgb(61, 56, 70);
border-radius:20px;
}
QProgressBar::chunk{
background-color: rgb(38, 162, 105);


border-radius:20px;
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:40px;
border:3px solid  red;
border-radius:20px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Storage Information</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:2px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="updatedate_but">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color: rgb(246, 245, 244);
border:3px solid black;
border-radius:20px;</string>
     </property>
     <property name="text">
      <string>Update Date And Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(255, 255, 255);
border-radius:20px;
font:bold;
border:1px solid black

</string>
     </property>
     <property name="text">
      <string>TIME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(255, 255, 255);
border-radius:20px;
font:bold;
border:1px solid black

</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(255, 255, 255);
border-radius:20px;
font:bold;
border:1px solid black

</string>
     </property>
     <property name="text">
      <string>SSID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(255, 255, 255);
border-radius:20px;
font:bold;
border:1px solid black

</string>
     </property>
     <property name="text">
      <string>PSK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color: rgb(255,255,255);
border-radius:20px;
border:2px solid black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color: rgb(255,255,255);
border-radius:20px;
border:2px solid black;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>440</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Existing Entries</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updatenetbut">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:3px solid black;</string>
     </property>
     <property name="text">
      <string>Update NetWorking </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>330</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="clockbut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:5px solid black;</string>
   </property>
   <property name="text">
    <string>Clock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 128, 137);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="storagebut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15pt &quot;Ubuntu&quot;;
background-color:rgb(255,255,255);
border-radius:20px;
border:5px solid black;</string>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbut">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/backarrowimg.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
